README.md

```diff
@@ -36,9 +36,9 @@ While we are strongly committed to piloting the use of Qubes OS for SecureDrop,
 
 Installing this project is involved. It requires an up-to-date Qubes 4.0 installation running on a machine with at least 12GB of RAM. You'll need access to a SecureDrop staging server as well.
 
-#### Qubes 4.0
+#### Qubes 4.0.1
 
-Before trying to use this project, install [Qubes 4.0](https://www.qubes-os.org/downloads/) on your development machine. Accept the default VM configuration during the install process.
+Before trying to use this project, install [Qubes 4.0.1](https://www.qubes-os.org/downloads/) on your development machine. Accept the default VM configuration during the install process.
```

Comment:
In order to mitigate against CVE-2019-3462, Qubes recommends installing all debian-based templates from scratch. Shipping out own Debian-based template should help/mitigate for all non-whonix based templates

https://www.qubes-os.org/news/2019/01/23/qsb-46/

Reply:
My understanding is that thanks to [this patch](https://github.com/QubesOS/qubes-manager/pull/149/files), upgrading the debian-9 template in-place on first boot of the workstation will handle the case appropriately. You're right that you should add logic to verify it's done, to guard against mistakes. Hopefully Qubes 4.0.2 will back in the latest apt version in the Debian templates, but no reason to expect that'll be in place by the beta.

Reply:
We opened issue #249 to track.